src/App.js

```diff
@@ -1,25 +1,38 @@
 import logo from './logo.svg';
 import './App.css';
+import Navbar from './components/Navbar';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import { Route } from 'react-router-dom';
+import countries_data from './countries.json';
+import React, { Component } from 'react'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default class App extends Component {
+  state = {
+    countries: []
+  };
 
-export default App;
+  componentDidMount() {
+    this.setState({
+      countries: countries_data
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <Navbar />
+          <div className='container'>
+            <div className='row'>
+              <div className='col-5'>
```

Comment:
You could add the following style to keep the country details visible on the right side:

```
max-height: 90vh;
overflow: scroll;
```